### DISCUSSION TITLE
"Introduction to window.ethereum" video  (Internal JSON-RPC Error)

### DISCUSSION THREAD

**Discussion by AlimGlck** [Cyfrin/foundry-full-course-cu#2702] [Q&A]
I get an Internal JSON-RPC Error``` when I try to fund 0.1 ETH.

![image](https://github.com/user-attachments/assets/11b1698f-d461-4544-b5ae-e5cf1498c8f3)

Have any of you guys encountered this problem before?

**Comment by EngrPips** [comment]
I did that section, and I didn't run into this problem. Have you confirmed that you have enough funds in your wallet?

**Reply by AlimGlck** [reply]
No, it is not that if it is it would say "insufficient gas"

**Reply by AlimGlck** [reply]
It is about the network I created I guess bc when I am connected to anvil it gives that error but when I am on Sepolia it does perfectly what I want to do.

**Reply by EngrPips** [reply]
Interesting. Maybe you should take time to reason and look through the code carefully to see if you can catch what you are doing wrongly.

**Reply by AlimGlck** [reply]
It isn't about code since it works on Sepolia which wouldn't work on Sepolia too if it is about code. But there is an interesting thing I noticed which is chain ID. The chain ID we are using is ``` 31337 ``` and this chain is also the chain ID of GoChain Testnet so I am not sure but is there a possibility that it broke funding?

**Reply by EngrPips** [reply]
I mean, I trust that you can figure this out. You should look into what you suspect, but I did the section, and I didn't run into any problem whatsoever.
